tests/BaseTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+use Illuminate\Foundation\Application;
+use Illuminate\Support\ServiceProvider;
+use Momento\Cache\Errors\UnknownError;
+use Momento\Cache\MomentoServiceProvider;
+use Momento\Cache\MomentoStore;
+
+abstract class BaseTest extends Orchestra\Testbench\TestCase
+{
+    /**
+     * Get package providers.
+     *
+     * @param Application $app
+     *
+     * @return array<int, class-string<ServiceProvider>>
+     */
+    protected function getPackageProviders($app)
+    {
+        return [
+            MomentoServiceProvider::class
+        ];
+    }
+
+    /**
+     * Setup the test environment.
+     */
+    protected function setUp(): void
+    {
+        if (!getenv('MOMENTO_CACHE_NAME')) {
+            throw new UnknownError("An environment variable named MOMENTO_CACHE_NAME must be set.");
+        } else if (!getenv('MOMENTO_AUTH_TOKEN')) {
+            throw new UnknownError("An environment variable named MOMENTO_AUTH_TOKEN must be set.");
```

Comment:
Kind of a minor nitpick, but if a user doesn't have either environment variable set, they'll run into two errors before they get straightened out. It might be slightly more helpful to check if either one is missing and, if so, tell the user about both variables in the error message.